symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/service/SymphonyBdkHealthIndicator.java

```diff
@@ -55,18 +54,16 @@ protected void doHealthCheck(Health.Builder builder) throws Exception {
     V3HealthStatus dfStatus = services.get(DF).getStatus();
     V3HealthStatus kmStatus = services.get(KM).getStatus();
     V3HealthStatus agtStatus = users.get(AGT).getStatus();
-    V3HealthStatus ceStatus = users.get(CE).getStatus();
     V3HealthStatus datafeedLoop = healthService.datafeedHealthCheck();
 
     boolean global = podStatus == V3HealthStatus.UP && dfStatus == V3HealthStatus.UP && kmStatus == V3HealthStatus.UP
-        && agtStatus == V3HealthStatus.UP && ceStatus == V3HealthStatus.UP && datafeedLoop == V3HealthStatus.UP;
+        && agtStatus == V3HealthStatus.UP && datafeedLoop == V3HealthStatus.UP;
 
     builder.status(global ? Status.UP.getCode() : "WARNING")
         .withDetail(POD, services.get(POD))
         .withDetail(DF, services.get(DF))
         .withDetail(KM, services.get(KM))
         .withDetail(AGT, users.get(AGT))
-        .withDetail(CE, users.get(CE))
```

Comment:
do we hide details about CE  too? Maybe we can keep details and only ignore CE in the global status